modules/ROOT/partials/misc/inline-formatting-of-list-bullets.adoc

```diff
@@ -0,0 +1,15 @@
+== Inline formatting.
+
+List bullets and list numbers can be spot- or inline-formatted.
```

Comment:
Is `spot-formatting` something we've used elsewhere?

Reply:
With no context as to what "spot-formatting" is/was, I read the initial statement as a branching of two types of formatting:

* spot
* inline

Then, a few paragraphs down, I decided they must just be alternate names for the same thing. Can we make this clearer?

Reply:
Added an explanation of what inline-formatting (aka spot-formatting) is.